components/footer/footer.tsx

```diff
@@ -0,0 +1,123 @@
+import Image from 'next/image';
+import styles from '../../styles/Home.module.css';
+import { classnames } from 'tailwindcss-classnames';
+
+const Footer = (): JSX.Element => {
+  return (
+    <footer className={classnames('bg-charcoal', 'flex', 'text-white', 'justify-center', 'py-12')}>
```

Comment:
We should probably make a container component so we can make sure the X margins on this match across the site.

Reply:
Well, I should have looked at the Trello board first. I am making this now. I'll add it to the footer if this PR goes in first.

Reply:
Ty for the container update -- and re--content/links - yeah, they're omitted. I asked Ders about what we wanted for content and I understood it as the Need Help image/link and then socials/copyright info from the live site.